### PR TITLE
Add Bill

### PR DESCRIPTION
**Pull request by 40504**

### CODE REVIEW

File: _data/payments.yml [hunk 1]
```diff
@@ -24,6 +24,15 @@ websites:
...
       - us
 
+  - name: Bill
+    url: https://www.bill.com
+    img: bill.png
+    tfa:
+      - sms
+      - phone
+      - totp
+    doc: https://help.bill.com/hc/en-us/articles/360000023843
```
Comment (phallobst) [inline _data/payments.yml:34]:
```suggestion
    doc: https://help.bill.com/hc/en-us/articles/360000023843
    exception: "2FA via phone call is available for US and CA only."
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Comment by 40504** [comment]
how to find in which countries the service is available?

**Comment by phallobst** [comment]
On the doc page, it says so in the section "Use an international phone number for MFA/2-step verification":
> Codes via phone call are only available to cell phone numbers from the US and Canada.

**Review by phallobst — APPROVED** [review]

**Review by ApCoder123 — CHANGES_REQUESTED** [review]
Please change your icon to an SVG:

**Review by ApCoder123 — APPROVED** [review]

**Pull request merged**